core/dbt/adapters/sql/impl.py

```diff
@@ -9,6 +9,7 @@
 from dbt.adapters.sql import SQLConnectionManager
 from dbt.logger import GLOBAL_LOGGER as logger
 
+from core.dbt.adapters.factory import BaseRelation
```

Comment:
```suggestion
from dbt.adapters.factory import BaseRelation
```

Reply:
What's the difference between core and non-core?

Reply:
dbt makes use of "namespace packages", as defined in [PEP 420](https://www.python.org/dev/peps/pep-0420/). The folder `core` contains one root namespace package in the `dbt` namespace, which you can install as `dbt-core`. The various plugin folders also contain root namespace packages in the `dbt` namespace - in particular they modify `dbt.include` and `dbt.adapters` by adding appropriate entries there.

So `core` is part of the folder structure and doesn't have any semantic meaning on the python side.

Reply:
Thanks for the explanation, appreciate it.